indico/modules/search/schemas.py

```diff
@@ -19,6 +19,11 @@
 from indico.web.flask.util import url_for
 
 
+class PlaceholderSchema(mm.Schema):
+    key = fields.String()
+    label = fields.String()
```

Comment:
shouldn't they be `required=True`?

Reply:
Yes, I didn't even remember they weren't by default